<launch>
  <arg name="data_path" default="/var/data/kitti/dataset/"/>
  <arg name="left_topic" default="/kitti/left_image"/>
  <arg name="right_topic" default="/kitti/right_image"/>
  <arg name="left_color_topic" default="/kitti/left_color_image"/>
  <arg name="right_color_topic" default="/kitti/right_color_image"/>

  <!-- Node -->
  <node pkg="kitti_player" type="kitti_player" name="kitti_player">
    <param name="data_path" type="string" value="$(arg data_path)"/>
    <param name="left_topic" type="string" value="$(arg left_topic)"/>
    <param name="right_topic" type="string" value="$(arg right_topic)"/>
    <param name="left_color_topic" type="string" value="$(arg left_color_topic)"/>
    <param name="right_color_topic" type="string" value="$(arg right_color_topic)"/>
  </node>
</launch>
